platform/telemetry/test/TelemetryFormatterSpec.js

```diff
@@ -57,6 +57,7 @@ define(
             });
 
             it("formats ranges as values", function () {
+                // eslint-disable-next-line no-loss-of-precision
```

Comment:
@trusktr Can you take a look at look at what Dave is asking here?

Reply:
My intent was not to change any runtime behavior.

However, I've since updated the base update-eslint PR, and I re-made this PR, and `lint:fix` passed without me needing to add any comment or change any code.